TTS/auto_tts/complete_recipes.py

```diff
@@ -0,0 +1,159 @@
+from TTS.auto_tts.model_hub import TtsModels, VocoderModels
+from TTS.auto_tts.utils import data_loader
+from TTS.trainer import Trainer, TrainingArgs, init_training
+
+
+class TtsExamples:
+    """This is trainer for calling complete recipes based off public datasets.
+    all configs are based off pretrained model configs or the model papers.
+
+    usage:
```

Comment:
`Examples:` instead of `usage:` 

It'd be nice to add `Args:`  and type annotations in docstrings too